crates/cubecl-core/src/runtime_tests/cmma.rs

```diff
@@ -42,6 +42,41 @@ pub fn kernel_simple_1(lhs: &Array<f16>, rhs: &Array<f16>, out: &mut Array<f32>)
     cmma::store(out.as_slice_mut(), &c, 16, cmma::MatrixLayout::RowMajor);
 }
 
+#[cube(launch)]
+/// Executes Out = Lhs @ Rhs.T
+pub fn kernel_simple_tf32<F: Cast>(lhs: &Array<F>, rhs: &Array<F>, out: &mut Array<f32>) {
+    let a = cmma::Matrix::<F>::from_slice(
+        cmma::MatrixIdent::A,
+        16,
+        16,
```

Comment:
The function is generic over `F`, I think for clarity it would be better to hardcode `tf32`.

Reply:
The problem is that there's some weirdness with trait resolution which made it not work using `tf32` directly for some reason. So I had to make it generic.